docs/using/installing/index.rst

```diff
@@ -219,15 +225,15 @@ Using Rackspace
 Another way to get a Flocker cluster running is to use Rackspace.
 You'll probably want to setup at least two nodes.
 
-#. Create a new Cloud Server running Fedora 20
+#. Create a new cloud server:
 
    * Visit https://mycloud.rackspace.com
    * Click "Create Server".
-   * Choose the Fedora 20 Linux distribution as your image.
+   * Choose a supported Linux distribution as your image (For available distributions, see the list of installation instructions in Step 3).
```

Comment:
"(For ...)" is a continuation of the sentence so should not be a capital letter as it does not proceed a full stop. I'd say just have a full stop.